apps/src/templates/studioHomepages/teacherHomepageV2/SectionList.tsx

```diff
@@ -1,30 +1,79 @@
-import React from 'react';
+import React, {useState} from 'react';
 
+import {removeSectionOrThrow} from '@cdo/apps/templates/teacherDashboard/teacherSectionsRedux';
 import {SectionMap} from '@cdo/apps/templates/teacherDashboard/types/teacherSectionTypes';
-import {useAppSelector} from '@cdo/apps/util/reduxHooks';
+import {useAppSelector, useAppDispatch} from '@cdo/apps/util/reduxHooks';
+import i18n from '@cdo/locale';
 
 import {SectionCard} from './SectionCard';
+import {SectionDeleteModal} from './SectionDeleteModal';
 
 import styles from './teacherHomepage.module.scss';
 
 interface SectionListProps {
   showHiddenOnly: boolean;
 }
 
+const NO_SECTION_ID: number = -1;
```

Comment:
```suggestion
const NO_SECTION_ID = -1;
```
I don't *think* you need to specify the type here because it's a const.